notification/src/interfaces/transformationResults/transformationEventInterface.ts

```diff
@@ -0,0 +1,22 @@
+
+import JobResult from './jobResult';
+
+/**
+ * Event send by the Transformation service upon transformation finish
+ */
+export interface TransformationEventInterface {
+    pipelineID: number      // ID of the pipeline
```

Comment:
Please use `Id` (lower camel case) instead of `ID` for consistency through the whole ODS